report/trustymail_report.mustache

```diff
@@ -360,7 +359,7 @@ opacity=0.04
 \label{sec:executive summary}
 Email underpins interactions between government agencies and enables direct correspondence with US citizens, immigrants, and international users. However, email was not built with security in mind, and without supplementary configuration or architecture it lacks important properties of trusted communication. Several email authentication technologies work in harmony to enable a recipient to detect and mitigate fraudulent email, particularly SPF and DMARC.
 
-This weekly report, created by the National Cybersecurity Assessments and Technical Services (NCATS) team inside the \href{https://www.cisa.gov/}{Cybersecurity and Infrastructure Security Agency (CISA)}, is intended to assist your agency to comply with the email security aspects of \href{https://cyber.dhs.gov}{Binding Operational Directive (BOD) 18-01}. Implementation guidance can be found at \href{https://cyber.dhs.gov/bod/18-01}{https://cyber.dhs.gov/bod/18-01/\#compliance-guide}.
+This weekly report, created by \href{https://www.cisa.gov/}{Cybersecurity and Infrastructure Security Agency (CISA)}, is intended to assist your agency to comply with the email security aspects of \href{https://cyber.dhs.gov}{Binding Operational Directive (BOD) 18-01}. Implementation guidance can be found at \href{https://cyber.dhs.gov/bod/18-01}{https://cyber.dhs.gov/bod/18-01/\#compliance-guide}.
```

Comment:
Should the "the" preceding CISA's full name have been removed? It reads awkwardly in non-acronym form.

Reply:
You are correct @mcdonnnj - thanks for catching that!  I didn't mean to remove the "the".  Fixed in https://github.com/cisagov/trustymail_reporter/pull/38/commits/5812ce3c418e4e9c56c909a82864db0c069b33b8.

This is why we have reviews. 👍